app/queries/content_items_query.rb

```diff
@@ -3,14 +3,21 @@ def self.build(options)
     ContentItemsQuery.new.results(options)
   end
 
-  def results(options = {})
+  def self.build_paged(options)
+    ContentItemsQuery.new.results(options, true)
+  end
+
+  def results(options = {}, should_page = false)
```

Comment:
I would rename this method to `build` just to be consistent with `build_paginated`.
On the other side, what about using named parameters this way?:

```ruby
def results(options = {}, should_page: false)
```

It will be benefitical as calling the method will read a bit better.